publications/the-software-craftsman.html

```diff
@@ -0,0 +1,156 @@
+---
+layout: default
+title: The Software Craftsman
+---
+
+<div class="breadcrumbs-v3 thesoftwarecraftsman">
+	<div class="container text-center"> 
+		<h1>{% t the-software-craftsman.title %}</h1>
+	</div>
+</div>
+
+<div class="container content-md">
+
+	<div class="row">
+		
+  	</div>
+
+  	<div class="row">
+	  	<div class="container">
+	  		<div class=" md-col-10">
+		      	<p>Published in December 2014 by our co-founder Sandro Mancuso, The Software Craftsman defines the Software Craftsmanship ideology and what it means to be a professional software developer.</p>
+	      	</div>
+	  	</div>
+	</div>
+
+  	<div class="row">
+  		<div class="container">
+  			<div class="md-col-6">
+				<div class="quote-v2">
+				<p>This book is an encyclopedia on the behavior, attributes, and structure of an organization striving to grow in professionalism and adhere to the principles of Software Craftsmanship.</p>
+				<span>Robert C. Martin</span>
+			</div>
+			</div>
+		</div>
+	</div>	
+
+  	<div class="row">
+  		<div class="container md-col-6">
+	  		<p>The book covers a wide range of subjects related to our profession and is full of advices and personal stories which illustrate the current state of our industry, how things could be better, and what developers can do to bring more professionalism, pragmatism, and pride to our industry.</p>
+	  		<p>The book clearly resonates with Robert C. Martin who has provided a beautifully written foreword. He summarises the book as an <span>encyclopedia on the behaviour, attributes, and structure of an organization striving to grow in professionalism and adhere to the principles of Software Craftsmanship.</span></p>
+	  		<p>The preface has a very inspirational story about how Sandro encountered his first mentor and how that relationship has shaped his personal and professional life.</p>
+  		</div>
+	</div>	
+
+	<div class="row">
+		<div class="title-header">
+      		<h2>Part I - Ideology and Attitude</h2>
+		<p>In the first part, Sandro defines the Software Craftsmanship ideology and the attitude expected from true software craftsmen.</p>
+    	</div>
+  	</div>
+
+  	<div class="row">
+		<div class="container col-md-6 col-sm-6">
+			<ul class="list-inline">
+				<li>
+					<span><b>Chapter 1 </b> Software Development in the Twenty-First Century</span>
+					<p>Describes how wrongly seniority is measured in most companies and explains how developers should evolve and behave in order to cope with more modern ways of working.</p>
```

Comment:
Describes how seniority is measured the wrong way in most companies...
... in order to work in a modern environment